features/nanostack/coap-service/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(mbed-os
+	PUBLIC
+		${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
please fix alignment here and also line 13 (tabs?)

Reply:
There were some tabs (->) instead of spaces. Changing that now.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/15fbe3d02fb329bf4c5bf77330243dd3c841c35a..7807ad10b5b9f726d9d77fe8c04bc2c20392e49e).